vsts/generate_javadocs_branch.ps1

```diff
@@ -104,10 +114,26 @@ function CreateJavadocReleaseBranch($GitHubName, $GitHubEmail, $Sources, $Folder
 
     # Move the generated content to the correct folder. The folder will be different for master and preview branches.
     New-Item -Path ../$FolderName -ItemType Directory
-    Move-Item -Force -Path ..\deps -Destination ..\$FolderName\deps
-    Move-Item -Force -Path ..\device -Destination ..\$FolderName\device
-    Move-Item -Force -Path ..\service -Destination ..\$FolderName\service
-    Move-Item -Force -Path ..\provisioning -Destination ..\$FolderName\provisioning
+
+    if($UpdateDepsDocs)
+    {
+        Move-Item -Force -Path ..\deps -Destination ..\$FolderName\deps
+    }
+
+    if($UpdateDeviceClientDocs)
+    {
+        Move-Item -Force -Path ..\device -Destination ..\$FolderName\device
+    }
+
+    if($UpdateServiceClientDocs)
+    {
+        Move-Item -Force -Path ..\service -Destination ..\$FolderName\service
+    }
+
+    if($UpdateProvisioningClientDocs)
```

Comment:
There are several clients within the provisioning folder, so this could be more specific. We have provisioning device client, provisioning service client, security provider, etc.

Reply:
Probably not a dealbreaker though

Reply:
Makes sense. I'm hoping we won't release those in preview :P There will be too many variables to deal with so I did not add them all. If we really need it, we can add it. Let me know if that sounds reasonable.